bindings/rust/wasmedge-sdk/examples/quickstart.rs

```diff
@@ -2,23 +2,23 @@
 
 #[cfg_attr(test, test)]
 fn main() -> Result<(), Box<dyn std::error::Error>> {
-    let module_path = std::path::PathBuf::from(env!("WASMEDGE_DIR"))
-        .join("tools/wasmedge/examples/fibonacci.wasm");
+    // let module_path = std::path::PathBuf::from(env!("WASMEDGE_DIR"))
```

Comment:
Ditto.

Reply:
Accepted. Will add a comment for it. Thanks!